modin/backends/pyarrow/parsers.py

```diff
@@ -11,13 +11,43 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""Module houses Modin parser classes, that are used for data parsing on the workers."""
+
 from modin.data_management.utils import get_default_chunksize
 from io import BytesIO
 import pandas
 
 
 class PyarrowCSVParser:
-    def parse(self, **kwargs):
+    """Class for handling CSV files on the workers using PyArrow backend."""
+
+    def parse(self, **kwargs):  # noqa: PR02
```

Comment:
can we write docstrings which don't need error checks turned off? if we cannot do so now please add a `FIXME` comment explaining the reason for this `noqa` thing.

Reply:
We can't rewrite this doc-string without `noqa`, because `**kwargs` are hiding actual function arguments. Spreading this `**kwargs` is really straightforward, so I did it at the last commit instead of adding `FIXME`, as I did in the #2956 by Devins [suggestion](https://github.com/modin-project/modin/pull/2956#discussion_r628795292).